vendor/bat-native-ads/src/bat/ads/internal/account/issuers/issuers_util.cc

```diff
@@ -50,15 +51,20 @@ void SetIssuers(const IssuersInfo& issuers) {
   AdsClientHelper::GetInstance()->SetIntegerPref(prefs::kIssuerPing,
                                                  issuers.ping);
 
-  ConfirmationStateManager::GetInstance()->SetIssuers(issuers.issuers);
-  ConfirmationStateManager::GetInstance()->Save();
+  AdsClientHelper::GetInstance()->SetListPref(prefs::kIssuers,
+                                              IssuersToValue(issuers.issuers));
 }
 
 IssuersInfo GetIssuers() {
   IssuersInfo issuers;
+
   issuers.ping =
       AdsClientHelper::GetInstance()->GetIntegerPref(prefs::kIssuerPing);
-  issuers.issuers = ConfirmationStateManager::GetInstance()->GetIssuers();
+
+  const absl::optional<base::Value::List> list =
+      AdsClientHelper::GetInstance()->GetListPref(prefs::kIssuers);
+  DCHECK(list);
```

Comment:
I would prefer to do if statement here as prefs could be changed outside.

Reply:
+1 fixed